.github/workflows/main.yml

```diff
@@ -162,7 +162,9 @@ jobs:
       - name: Generate IDLs
         run: pnpm generate:idls
       - name: Ensure working directory is clean
-        run: test -z "$(git status --porcelain)"
+        run: |
+          git status --porcelain
+          test -z "$(git status --porcelain)"
```

Comment:
What exactly does duplicating this command do?

Reply:
It will log the differences. It will only produce an output when `test -z "$(git status --porcelain)"` fails, so it is useful to see "why" it failed.